src/Administration/Resources/app/administration/src/module/sw-sales-channel/view/sw-sales-channel-detail-base/sw-sales-channel-detail-base.html.twig

```diff
@@ -213,6 +213,14 @@
             </sw-card>
         {% endblock %}
 
+        {% block sw_sales_channel_detail_base_options_cookie %}
+            <sw-sales-channel-detail-cookie
```

Comment:
Imho it would be totally fine to put the `sw-switch-field` and the `sw-card` directly in this component and not creating a new component for this.